thrust/thrust/copy.h

```diff
@@ -284,7 +284,7 @@ template<typename InputIterator, typename Size, typename OutputIterator>
  *  ...
  *  struct is_even
  *  {
- *    __host__ __device__
+ *    _CCCL_HOST_DEVICE
```

Comment:
important: same note as for CUB, docs should not refer to implementation detail macro, otherwise it seems like we require users to write operators this way. There are many more places in thrust where implementation detail macro spilled into docs, please, return old code there as well.